sdk/keyvault/azure-security-keyvault-keys/test/ut/recordings/KeyVaultKeyClient.RecoverOperationResumeToken2Goqu.json

```diff
@@ -0,0 +1,187 @@
+{
```

Comment:
Is this new json file referenced anywhere (or relevant to this PR)? I don't see a new test case that leverages this.

Unless I am missing something, maybe exclude it from this PR.